src/wp-includes/customize/class-wp-customize-nav-menus-panel.php

```diff
@@ -98,4 +98,25 @@ protected function content_template() {
 		<li class="customize-control-title customize-section-title-nav_menus-heading"><?php _e( 'Menus' ); ?></li>
 		<?php
 	}
+
+	/**
+	 * Checks required user capabilities and whether the theme has the
+	 * feature support required by the panel.
+	 *
+	 * @since 4.0.0
```

Comment:
```suggestion
	 * @since 5.9.0
```

Reply:
Hmm, but hasn't this method always existed on `WP_Customize_Nav_Menus_Panel` by virtue of it existing gin the base class? 😀

Reply:
I'm not sure but let's go with `5.9.0` 😀